app/handlers/gate.go

```diff
@@ -15,19 +14,24 @@ import (
 )
 
 type gateHandler struct {
-	defaultTimeout    time.Duration
+	defaultTimeout    int64
 	defaultRules      []domain.Rule
 	adapterDispatcher *adapters.AdapterDispatcher
 	logger            *zap.Logger
 }
 
+type OpenRequestInput struct {
+	Ip      string `form:"ip" json:"ip" binding:"omitempty,ip"`
```

Comment:
🤔 this breaks passing a subnet? I can no longer whitelist entire ranges using `/16`